.prettierrc

```diff
@@ -6,10 +6,10 @@
     {
       "files": "*.md",
       "options": {
-        "printWidth": 70,
+        "printWidth": 80,
         "useTabs": false,
         "trailingComma": "none",
-        "proseWrap": "never"
+        "proseWrap": "always"
```

Comment:
Please let's not do this, it's super annoying and unmaintainable option ;-( It's also impossible to review this PR now.

Reply:
I can revert this in this PR but I also found it is counter intuitive that I need to change my editor's settings to be able to read long lines. I guess it's going to be the same for most of people. So I'd propose this as a change in another PR.

Why do you say it's "unmaintainable"? Having automatic formatting should work for everyone, no?

Reply:
1. Diffs are going to be broken: if you add a new word, you'll have changes in all lines of the paragraph after the added work, and it will be impossible to understand the actual change. With long lines it's clear.

2. Editing is painful even with autoformatting, because you have to constantly reformat the fail to keep it readable.

Reply:
I agree with both especially the diff argument. thanks for clarifying!